src/Frontend/src/components/messages2/SagaDiagram/SagaPluginNeeded.vue

```diff
@@ -7,11 +7,11 @@ import CopyToClipboard from "@/components/CopyToClipboard.vue";
   <div class="body" role="status" aria-label="saga-plugin-needed">
     <div class="saga-message">
       <div class="saga-message-container">
-        <img class="saga-message-image" :src="NoSagaIcon" alt="" />
-        <h1 class="saga-message-title">Saga audit plugin needed to visualize saga</h1>
+        <img class="saga-message-image" :src="NoSagaIcon" alt="" v-tippy="`Saga plugin required`" />
+        <h1 class="saga-message-title" v-tippy="`The saga audit plugin is required to show saga visualization`">Saga audit plugin needed to visualize saga</h1>
         <div class="saga-message-box">
-          <p class="saga-message-text">To visualize your saga, please install the appropriate nuget package in your endpoint</p>
-          <a href="https://www.nuget.org/packages/NServiceBus.SagaAudit" class="saga-message-link">install-package NServiceBus.SagaAudit</a>
+          <p class="saga-message-text" v-tippy="`You need to install the NServiceBus.SagaAudit package in your endpoint`">To visualize your saga, please install the appropriate nuget package in your endpoint</p>
+          <a href="https://www.nuget.org/packages/NServiceBus.SagaAudit" class="saga-message-link" v-tippy="`Go to NServiceBus.SagaAudit NuGet page`">install-package NServiceBus.SagaAudit</a>
```

Comment:
no need to tell what a link does
```suggestion
          <p class="saga-message-text">To visualize your saga, please install the appropriate nuget package in your endpoint</p>
          <a href="https://www.nuget.org/packages/NServiceBus.SagaAudit" class="saga-message-link">install-package NServiceBus.SagaAudit</a>
```